kroxylicious-operator-dist/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>0.14.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kroxylicious-operator-dist</artifactId>
+    <name>Kubernetes Operator distribution</name>
+    <description>Assembles the operator assets for release</description>
+    <packaging>pom</packaging>
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-operator</artifactId>
+            <version>${project.version}</version>
+            <scope>runtime</scope>
+        </dependency>
+    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>dist</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-assembly-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>operator-asset</id>
+                                <phase>package</phase><!-- Note this execution happens after the following plugins bound to the package phase! -->
+                                <goals>
+                                    <goal>single</goal>
+                                </goals>
+                                <configuration>
+                                    <appendAssemblyId>false</appendAssemblyId>
+                                    <finalName>kroxylicious-operator-${project.version}</finalName>
```

Comment:
Did you have any thoughts about the uber assert or one asset per installer e.g. yaml vs olm vs helm? Just wondering if we should include another qualifier in the `finalName`.

Reply:
I've been using the word asset refer to the things which get attached to the github release page. AFAIU OLM (which is not very far) when we support it, it won't require a downloadable artifact, but rather there's some kind of a store/repository mechanism which will allow people to install via the route. Maybe that's not true for Helm, but since there's not really a priority I'm inclined towards "let's not overthink this" for now.